### ISSUE TITLE
edit presets

### ISSUE THREAD

**Issue by MoBolls** [1280px/hueblocks-legacy#5]
it would be nice if there were a way to edit custom presets after they are created. that way you don't have to re-input all the blocks you want every time you want to remove one.

**Comment by nobu-sh** [comment]
I can PR this @1280px if you would like.
I've been looking into uploading entire folders too, however for safety reasons the browser acts kinda wacky about it :/

**Comment by 1280px** [comment]
The thing is... I want to re-write the way preset are working. Right now it's just a bunch pre-created .JS lists of blocks, while what I want is to somehow make them generate-able from version X palette combination (the loaded blockset will also be defined by selected version instead of manual switching, of course).

So I guess it's better to postpone this and #4 suggestions before the new preset system is created because current implementations most likely won't work with it.

**Comment by nobu-sh** [comment]
Yeah I noticed that, maybe it would be best to run a GH workflow or even a cron job to run on update to the game. 
You could easily just download the the server zip from bedrock edition and create a pull request with the updated textures.

**Comment by 1280px** [comment]
Back then I didn't know even a half of these smart words :))) Nowadays version JARs are automatically downloaded and processed on every push, although it still doesn't support bedrock (does it even make sense to support it, considering it's always had the same textures as latest MCJE version?)

As of the issue, in HBN it is possible to both edit a custom-made palette **[context: at some point in 2023 or 2024 presets were renamed to palettes because I wanted to reduce the confusion between terms "blockset" and "preset"]**, or even select any pre-added preset and modify it for own needs. It can be done by first selecting a palette you need, and then selecting "Edit current palette..." option -- palette editor will automatically pre-select all textures the original palette contained (except for "All blocks" palette, which will not pre-select anything -- this is done by design)
